### DISCUSSION TITLE
Consistent changelogs?

### DISCUSSION THREAD

**Discussion by karfau** [pkgxdev/pkgx#663] [Help] [answered]
Some releases contain information about changes, some don't.
Is there any way to make is more consistent to make to able to follow along?

**Comment by mxcl** [comment, marked as answer]
We use the auto-generated changelog option and it sometimes works well and other times does not.

I'd love to do a better job, but we are well into the v0 phase of things right now. There are a lot of moving parts and a lot of our releases are to manage all those pieces. I release new stuff on this org 5-6 times a week.

Once we're more stable we will do release notes properly.
